basic-android-kotlin-compose-training-mars-photos-starter/app/src/main/java/com/example/marsphotos/ui/screens/realEstateScreen/RealEstateViewModel.kt

```diff
@@ -26,6 +27,17 @@ class RealEstateViewModel(
         getRealEstatePrice()
     }
 
+
+    var selectedPosition: Int by mutableIntStateOf(-1)
```

Comment:
Why didn't you make this variable private?

Reply:
@cipry13 I just modified it. I put the set private, because we want to be able to modify the variable only from inside the class.

Reply:
In my case, get could also be private, because I don't use the variable outside of the class.